### ISSUE TITLE
Autocomplets DrawerList  rerender elements on scroll on second opening

### ISSUE THREAD

**Issue by darkdubdeep** [dnbexperience/eufemia#622]
## 🐛 Bug Report

When you click on autocomplete field on the second time and open the DrawerList with options it scrolls worse than you open the DrawerList the first time. When I check the DrawerList component on the "Elements" tab in the Chrome dev tools I see that every "li" element rerenders when you scroll on the second opening.

Didn't found a solution in documentation, I tried setting up "cache_hash" with some string but it not helps

## To Reproduce

go to the https://eufemia.dnb.no/uilib/components/autocomplete/demos#update-data-dynamically-on-first-focus  

Click on the input field, try to scroll the list, choose an option, click on the input field and try to scroll again

link to a video with behavior reproduction https://drive.google.com/file/d/1tTrQocVDBQaQdiRvcn9ydNHED1sHAIgP/view

## Expected behavior

Smooth scroll of DrawerList  and possibility to switch of rerendering elements in DrawerList  on scroll

**Comment by tujoworker** [comment]
Hi! 

Thanks for the good points and leading towards an enhancement (and fixing a bug). The bug was, that it should have happened also on the very first try. But in order to solve the real problem – the slow scrolling issue – I made this arrow color change feature optional, by using a prop `enable_closest_observer`. Maybe we can have usage of that behaviour in sertain situations – or we may replace/remove it completely in future.

The reason why the scroll observer got included two years ago and was enabled by defualt, is the little visual detail (arrow color change ). I tried several solutions back then – and non have worked out well and flexible enought. But I'm confident we will come up with a better solutuion in future – thats why I have made this now as an opt in feature with that property.

What I'm talking about? ok, lett me explain; Once a "selected" item is on the very top (or bottom) – the arrow will change it's color – only by CSS. But in order to do so, a HTML class has to be set/removed once the item is at the exposed position. Have a look at the video:

![Dropdown arrow behaviour](https://user-images.githubusercontent.com/1501870/92084816-395f2400-edc8-11ea-9bb8-72009f0d7369.gif)
